src/js/media-progress-range.js

```diff
@@ -178,11 +180,11 @@ class MediaProgressRange extends MediaChromeRange {
         this.thumbnailPreview.style.left = `${thumbnailLeft}px`;
         this.thumbnailPreview.time = mousePercent * this.media.duration;
       };
-      window.addEventListener('mousemove', mouseMoveHandler, false);
+      Window.addEventListener('mousemove', mouseMoveHandler, false);
```

Comment:
If this did actually get called it would error because addEventListener isn't defined on the mock Window, but we're ok because UI events won't get triggered on the server. But then it's another place to question if we need the mock Window just for consistency.

Reply:
for consistency, yes